contracts/metadata/ERC1155OnChainMetadata.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.1;
+
+import "@openzeppelin/contracts-upgradeable/token/ERC1155/ERC1155Upgradeable.sol";
+import "./OnChainMetadata.sol";
+
+/**
+ * @title On-chain metadata for ERC1155Upgradeable,
+ * making quick and easy to create html/js NFTs, parametric NFTs or any NFT with dynamic metadata.
+ * @author Daniel Gonzalez Abalde aka @DGANFT aka DaniGA#9856.
+ * @dev Note that this approach doesn't follow the ERC1155Metadata standard.
```

Comment:
should we also follow this? what would it take to support?

Reply:
discussed this is approved ✅